Trie.h

```diff
@@ -127,6 +149,21 @@ int Trie::getParentIndex(std::string cubeString) {
     return cubeStateData[insertTrieNode].parentIndex;
 }
 
+int Trie::getParentIndex(int cubeNodeIndex) {
+    return cubeStateData[cubeNodeIndex].parentIndex;
+}
+
+std::string Trie::getLastMove(int cubeNodeIndex) {
```

Comment:
It would be better to just pad all moves to length 2, so that its easier to process. For a clockwise move, just add a trailing space, so "F " for example instead of "F", so that it matches the length of the counterclockwise move